tgui/packages/tgui/interfaces/ChemDispenser.js

```diff
@@ -0,0 +1,449 @@
+
+import { Fragment } from 'inferno';
+import { useBackend, useSharedState, useLocalState } from "../backend";
+import { truncate } from "../format.js";
+import { Button, NumberInput, Section, Box, Table, Tooltip, Icon, Tabs, Input, Modal } from "../components";
+import { Window } from "../layouts";
+
+export const titleCase = str => {
```

Comment:
comment on what this does?

Reply:
Replaced with text-transform: capitalize